server/routers/billingRouter.js

```diff
@@ -19,6 +27,16 @@ router.get('/', async (req, res) => {
         });
 });
 
-
+// Example POST request with Stripe
+// TODO: Create actual POST request
+router.post("/", (req, res) => {
```

Comment:
why is the route "/" for stripe?

Reply:
I have to manually format each file that I work on. For whatever reason `Format on Save` is not working. Updated in the next commit.

That's purely an example. Once other billing routes are added that can be modified, changed, deleted as necessary.